services/svc-pinger/test/ping.middleware.test.ts

```diff
@@ -0,0 +1,88 @@
+import { describe, it, expect, vi } from 'vitest'
+import { pingMiddleware } from '../src/middlewares/ping.middleware'
+import { Request, Response, NextFunction } from 'express'
+
+const urlProps: ISchemeProps = { scheme: urlScheme, hostType: "URL" }
+import { ipv4Props } from '../src/schemes/ip.scheme'
+import { ISchemeProps } from '../src/types/api.interfaces'
+import { urlScheme } from '../src/schemes/url.scheme'
+
+describe('pingMiddleware', () => {
```

Comment:
To avoid unnecessary duplication of tests, the middleware should be tested within the integration tests of corresponding endpoints